tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -14,7 +14,22 @@
 //
 //******************************************************************************
 
-#include "DrawingTest.h"
+#import "DrawingTest.h"
+#import <windows.h>
+#import "CGContextInternal.h"
+
+static woc::unique_cf<CGColorSpaceRef> rgbColorSpace(CGColorSpaceCreateDeviceRGB());
```

Comment:
We really shouldnt be initializing globals with CG graphics method.  You can use this inside a static function if you really need this static instance.